### PR TITLE
various fixes

### PR DESCRIPTION
**Pull request by jsfez**
- fix: list layouts
- fix(build): status wording
- feat(build): hide useless stable screenshot filter

### CODE REVIEW

File: apps/app/src/pages/Build/index.js [hunk 1]
```diff
@@ -376,24 +364,27 @@ const BuildContent = ({ ownerLogin, repositoryName, buildNumber }) => {
...
-            <Button
-              variant="neutral"
-              onClick={() => setShowStableScreenshots((prev) => !prev)}
-              justifyContent="start"
-            >
-              <IllustratedText
-                icon={showStableScreenshots ? EyeClosedIcon : EyeIcon}
-                field
+            {stats.stableScreenshotCount > 0 &&
+            stats.stableScreenshotCount < pageInfo.totalCount ? (
```
Comment (gregberge) [inline apps/app/src/pages/Build/index.js:368]: I don't understand the second condition.

File: apps/app/src/pages/Build/index.js [hunk 2]
```diff
@@ -267,39 +267,27 @@ function StableScreenshots({ ownerLogin, repositoryName, buildNumber }) {
...
       stats,
       screenshotDiffs: { pageInfo, edges: screenshotDiffs },
     },
-  } = data.repository;
+  } = repository;
+
+  const { observe, inView } = useInView();
+
+  const [showStableScreenshots, setShowStableScreenshots] = React.useState(
+    pageInfo.totalCount === 0
```
Comment (gregberge) [inline apps/app/src/pages/Build/index.js:289]: Don't understand.

File: apps/app/src/pages/Build/index.js (hunk 1 above)
Comment (jsfez) [reply apps/app/src/pages/Build/index.js:368]: If there is only stable screenshots we set the filter to `falseˋ and hide the toggle button

File: apps/app/src/pages/Build/index.js (hunk 2 above)
Comment (jsfez) [reply apps/app/src/pages/Build/index.js:289]: See above.

File: apps/app/src/pages/Build/index.js (hunk 1 above)
Comment (gregberge) [reply apps/app/src/pages/Build/index.js:368]: ok

**Review by gregberge — APPROVED** [review]

**Pull request merged**